SPECS/k3s/k3s.spec

```diff
@@ -82,16 +82,10 @@ exit 0
 %{install_sh}
 
 %changelog
-* Tue Apr 11 2023 Anuj Garg <anujgarg@microsoft.com> - 1.26.2-2
-- Addressed CVE for containerd and changed it to the new patched version 1.6.18
-
-* Wed Apr 05 2023 Anuj Garg <anujgarg@microsoft.com> - 1.26.2-1
-- Updated k3s to 1.26.2
+* Mon Apr 17 2023 Anuj Garg <anujgarg@microsoft.com> - 1.26.3-1
+- Updated k3s to 1.26.3
 - Drop version constraint on golang <= 1.18.8
 
-* Wed Apr 05 2023 Anuj Garg <anujgarg@microsoft.com> - 1.25.5-1
-- Updated k3s to 1.25.5
-
```

Comment:
Why are we dropping these changelog entries?

Reply:
Dropped these change logs because they have been replaced with the new version changes so the older change logs will not be valid that's why removed.

Reply:
Ah, the other versions there haven't been published, I see. That's fine then.